src/slider.jsx

```diff
@@ -559,10 +560,10 @@ const Slider = React.createClass({
           onMouseLeave={this.handleMouseLeave}
           onMouseUp={this.handleMouseUp}
         >
-          <div ref="track" style={this.prepareStyles(styles.track)}>
-            <div style={this.prepareStyles(styles.filled)}></div>
-            <div style={this.prepareStyles(remainingStyles)}></div>
-            <div style={this.prepareStyles(handleStyles)} tabIndex={0} {...handleDragProps}>
+          <div ref="track" style={prepareStyles(styles.track)}>
+            <div style={prepareStyles(styles.filled)}></div>
+            <div style={prepareStyles(remainingStyles)}></div>
+            <div style={prepareStyles(handleStyles)} tabIndex={0} {...handleDragProps}>
```

Comment:
I can stick with your first approach, since the last one will initialise the variable with *false* or an *object*